parts/k8s/cloud-init/artifacts/cse_config.sh

```diff
@@ -114,7 +114,15 @@ configureChrony() {
 ensureChrony() {
   systemctlEnableAndStart chrony || exit {{GetCSEErrorCode "ERR_SYSTEMCTL_START_FAIL"}}
 }
-
+disable1804SystemdResolved() {
+  ls -ltr /etc/resolv.conf
```

Comment:
I assume these commands helped during debugging? Do we want to include anything other than the symbolic link statement in the final CSE implementation?

Reply:
I assume the same, I do not mind either way, will remove.